pkg/urlfetch/schemes.go

```diff
@@ -52,7 +52,7 @@ var (
 	DefaultHTTPClient = NewHTTPClient(http.DefaultClient)
 
 	// DefaultTFTPClient is the default TFTP FileScheme.
-	DefaultTFTPClient = NewTFTPClient()
+	DefaultTFTPClient = NewTFTPClient(tftp.ClientMode(tftp.ModeOctet), tftp.ClientBlocksize(1500), tftp.ClientWindowsize(65535))
```

Comment:
Maybe not 1500 for the blocksize.  RFC mentions both 1468 and 1456.  But you might get fragments when you add on the UDP/IP headers.

1500 is the max payload size handed to ethernet.  udp=8, ipv6=40.  so maybe 1452.

Reply:
I am going to go conservative and go 1450 :-)

In local tests on my laptop this reduced download of an 8M file from 16s to 5s.

Reply:
fantastic!